features/frontend/cart_promotions_fixed.feature

```diff
@@ -92,6 +93,20 @@ Feature: Checkout fixed discount promotions
           And "Promotion total: -€15.00" should appear on the page
           And "Grand total: €110.00" should appear on the page
 
+    Scenario: Most expensive promotion is applied when the cart has at
+              least one product
+        Given promotion "most expensive item" has following actions defined:
+            | type           | configuration |
+            | Fixed discount | Amount: 10    |
+          And promotion "most expensive item" has following rules defined:
+            | type     | configuration                      |
+            | item_most_expensive |  |
+        Given I am on the store homepage
+         When I add product "Potato" to cart, with quantity "1"
+         Then I should be on the cart summary page
+          And "Grand total: €190.00" should appear on the page
```

Comment:
All of the scenario are testing "promotion total". Consider adding it here as well.
What about a scenario with 2 items with the same price, what about scenario with 2 items with different price?